test/test_losses.py

```diff
@@ -9,6 +9,71 @@
 from torch.testing import assert_allclose
 
 
+class TestFocalLossWithLogits:
+    def test_smoke_none(self, device):
```

Comment:
can you add `dtype` in the signature as well ?
`def test_smoke_none(self, device, dtype):`
and then create the input tensor with that type